Header header
uint32 id
float64 yaw
float64 pitch
float64 roll